optimum/neuron/utils/hub_neuronx_cache.py

```diff
@@ -185,10 +234,14 @@ def download_file_to_string(self, filename: str, limit: int = None):
 
 def get_hub_cache():
     HUB_CACHE = "aws-neuron/optimum-neuron-cache"
-    return os.getenv("CUSTOM_CACHE_REPO", HUB_CACHE)
+    custom_hub_cache = load_custom_cache_repo_name_from_hf_home()
+    if custom_hub_cache is not None and len(custom_hub_cache) > 0:
+        return custom_hub_cache
+    else:
+        return os.getenv("CUSTOM_CACHE_REPO", HUB_CACHE)
```

Comment:
We already have a function that does all of this in `optimum/neuron/cache_utils.py` called `get_hf_hub_cache_repos`. I plan to "merge" the utils from this file with the `hub_neuronx_cache` utils that are in common, but just wanted to let you know.

Reply:
That will be awesome!! Yeah I also used utils from cache_utils, will be nice if we can organize them better.